src/components/PostEditor.js

```diff
@@ -302,12 +308,13 @@ export class PostEditor extends React.Component {
       editingTagDescriptions
     } = this.props
     const { currentUser } = this.context
-    const selectableTags = compact([this.props.tag].concat(specialTags))
+    const selectableTags = compact([this.props.tag, post.tag].concat(specialTags))
```

Comment:
actually, shouldn't this be `tag` as declared in the next line from `postEdit`, not `post`?